### ISSUE TITLE
Check accessibility of {}

### ISSUE THREAD

**Issue by DCEW** [nhs-r-community/NHSR-way#189]
From ZT: "This does need to be changed to remove {} as it has been mentioned in other code reviews that it's not easy to read/screen read when {} are used. I haven't had a chance to check this will any accessibility documentation but it might be good to move to back ticks for packages and functions in a style?"

**Comment by Lextuga007** [comment]
It was mentioned by @TimTaylor in relation to screen readers.

**Comment by Lextuga007** [comment]
@TimTaylor Thanks for replying so quickly 😄 I've also asked on the accessibility channel on the UK Gov Digital Slack as that's an active channel.
